src/mqt/predictor/ml/predictor.py

```diff
@@ -35,12 +35,31 @@
 class Predictor:
     """The Predictor class is used to predict the most suitable quantum device for a given quantum circuit."""
 
-    def __init__(self, logger_level: int = logging.INFO) -> None:
-        """Initializes the Predictor class."""
+    def __init__(
+        self,
+        figure_of_merit: reward.figure_of_merit,
```

Comment:
Might it be worth to add a default value here? expected fidelity would be the first guess.

Reply:
True, I followed your suggestion.